test/nyc-integration.js

```diff
@@ -193,9 +193,10 @@ t.test('interprets first args after -- as Node.js execArgv', t => testSuccess(t,
   args: ['--', '--expose-gc', path.resolve(fixturesCLI, 'gc.js')]
 }))
 
-t.test('--show-process-tree displays a tree of spawned processes', t => testSuccess(t, {
-  args: ['--show-process-tree', process.execPath, 'selfspawn-fibonacci.js', '5']
-}))
+// Disabled; see https://github.com/istanbuljs/nyc/pull/1546
+// t.test('--show-process-tree displays a tree of spawned processes', t => testSuccess(t, {
+//   args: ['--show-process-tree', process.execPath, 'selfspawn-fibonacci.js', '5']
+// }))
```

Comment:
As desired, this test is now commented out. I left a link back to this PR for any future spelunkers.